### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by enrospv** [Joshimuz/mcbingo#91]
Make the bingo translatable by volunteer users so non-English speaking people can use it aswell. I could help by also providing a Spanish translation.

I understand that this might not be a good idea yet because the bingo is constantly being updated with new goals and such, but I think it is a very good long-term investment. Minecraft itself is in many languages so it's very likely that someone would be interested on having this bingo in their own language.

You could add a button (inside the layout or on the webpage) to change the display language and it can try to detect the browser's language, defaulting to English.

**Comment by Joshimuz** [comment]
Yeah this was a feature I've always wanted to add and I have an idea on how to do it.
For the problem of changing goals all the time, I would only offer translations on completed versions and not the dev version which would fix that issue

I will get to this eventually
